lib/htmltoword/document.rb

```diff
@@ -145,7 +143,7 @@ def inject_image_content_types(source)
 
       #inject missing extensions into document
       missing_exts.each do |ext|
-        doc.at_css("Types").add_child( "<Default Extension='#{ext}' ContentType='image/#{ext}'/>")
+        doc.at_css("Types").add_child( "<Default Extension='#{ext}' ContentType='image/#{content_type_from_extension(ext)}'/>")
```

Comment:
I think the issue is simpler than that. It looks like I've just got the content type wrong:
https://msdn.microsoft.com/en-us/library/dd997170%28v=vs.100%29.aspx

That shows that 'jpg' or 'jpeg' should be 'jpg' whereas document.rb uses 'jpeg'.

Could something as simple as this fix the problem and not give rise to issues where the local file extension doesn't agree to the content type?

```
 #get extension from filename and clean to match content_types
 def ext_from_filename(filename)
   ext = File.extname(filename).delete(".").downcase

   case ext
   when "jpg", "jpeg" then "jpg"
   when "tif" then "tiff"
   else ext
   end
end
```